browser/ui/browser_commands.cc

```diff
@@ -1073,4 +1073,27 @@ void SwapTabsInTile(Browser* browser) {
                            /*select_after_move*/ false);
 }
 
+void ToggleSplitViewOrientation(Browser* browser) {
+  auto* split_view_data = SplitViewBrowserData::FromBrowser(browser);
+  if (!split_view_data) {
+    return;
+  }
+
+  if (browser->tab_strip_model()->empty()) {
+    return;
+  }
+
+  if (!IsTabsTiled(browser)) {
+    return;
+  }
+
+  auto* model = browser->tab_strip_model();
+  auto tab = model->GetActiveTab()->GetHandle();
```

Comment:
`GetActiveTab()` could give null. Is it safe w/o null checking?

Reply:
I believe so. We're checking tab strip model is empty up there, so it shouldn't happen.